app/views/payments/add.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Saldo opwaarderen voor #{@user.name}- SOFIA" %>
+
+<%= content_tag :div, id: 'payment-add', class: 'container footer-padding', data: {user: @user_json} do %>
+  <div class="container footer-padding">
+    <div class="row py-4">
+      <div class="col-sm-12">
```

Comment:
Just thinking, but maybe just `col` is enough here, since we're not providing alternatives for other screen sizes.